src/utils/__tests__/getNormalizedCountyData.test.js

```diff
@@ -22,6 +22,7 @@ describe("getNormalizedCountyData.test.js", () => {
   const mockCounty2 = "US_CO_ARAPAHOE";
   const mockCounty3 = "US_CO_ADAMS";
   const mockContyName = "Mock county";
```

Comment:
Nit: these should be `mockCountyName` and `mockCountyName1`

Reply:
Not exactly understand what you mean

Reply:
They are misspelled. Conty is a typo. County is not.

Reply:
I fixed it!